### PR TITLE
created landing page

### LINKED ISSUE #344: Create Entrepreneurs Landing Page

### PR DESCRIPTION
**Pull request by Arfah**
Closes #344 

Thanks

### CODE REVIEW

File: source/entrepreneurs.html.haml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+- description "Makers Academy is London's leading Web Developer Bootcamp. Learn to code in 12 Weeks by enrolling on one of our programming / web development courses! Learn Ruby, JavaScript and the best software practices."
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/dan-and-students-overhead-70.jpg")});"}
+  .container
+    .centered-row
+      %h1 Become your own cto
+      %h4.subheader learn to code in 12 weeks
+
+      %P Have you ever dreamed of starting your own business? Or maybe you're already a business owner and want to create your company's website yourself to ensure that your vision becomes a reality? Become involved in every aspect of your business and learn the most essential skill of the future: Coding! We've helped plenty of entrepreneurs like you take complete control over the technical aspects of their company and we can help do the same for you.
```
Comment (spike01) [inline source/entrepreneurs.html.haml]: lower case C in "coding"

Comment (spike01) [inline source/entrepreneurs.html.haml]: "We've helped plenty of entrepreneurs like you take complete control over the technical aspects of their company and we can help do the same for you" -> "We've helped many entrepreneurs take complete control over the technical aspects of their company and we can help do the same for you"

File: source/entrepreneurs.html.haml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      %h3 Why learn to code before launching your business?
+      %p You could have the greatest idea in your head but if you can’t execute it, it’s meaningless. Trying to find a technical co-founder to help build your idea is hard and outsourcing your development work is risky business.
+      %p In today's technology-driven world, knowing the ins and outs of web development could mean the difference between success and failure for companies. Save time, money and stress by learning how to build your own projects for your entrepreneurial ventures
+%hr
+
+%section.no-padding-top.no-min-height
+  .article--centered
+    %article
+      %h3 This could be you
+      %p Our graduates have gone on to build successful businesses and been accepted into accelerators and secured further funding. Read their stories below.
```
Comment (spike01) [inline source/entrepreneurs.html.haml]: "Our graduates have gone on to build successful businesses, been accepted into accelerators, and secured further funding. Read their stories below."

**Comment by spike01** [comment]
:shipit: with corrections

**Comment by Arfah** [comment]
:shipit: BOOM

**Pull request merged**